java/drivers/driver-hazelcast4plus/pom.xml

```diff
@@ -13,7 +13,7 @@
     </parent>
 
     <properties>
-        <hazelcast.version>5.4.0-SNAPSHOT</hazelcast.version>
+        <hazelcast.version>5.5.0-SNAPSHOT</hazelcast.version>
```

Comment:
needs to be 5.5.0

Reply:
aee8f32c27333889216b77ce3d60c92e9406a08f

Reply:
oops, looks like there were more changes in master that got merged into the PR -- need to scroll through them again